### PR TITLE
Structure of Gallery & Sitemap

### PR DESCRIPTION
**Pull request by pramodh123kit**
@RandilaP Structure of the Gallery and Sitemap

### CODE REVIEW

**Comment by RandilaP** [comment]
@pramodh123kit When you're adding a commit message please add a reasonable commit message like what you did, Like a title. And change the PR title too. And in the PR template please add some screenshots of what you did. it'll help the reviewer to understand what kind of changes you made. And good job @pramodh123kit thanks for getting it done

File: Gallery/styles.css [hunk 1]
```diff
@@ -0,0 +1,187 @@
+:root {
+  --red: #e20000;
+  --white: #ffffff;
+}
+
+
+html {
+  scroll-behavior: smooth;
+}
+
```
Comment (RandilaP) [inline Gallery/styles.css:9]: remove these unnecessary spaces.

File: Gallery/styles.css [hunk 2]
```diff
@@ -0,0 +1,187 @@
+:root {
+  --red: #e20000;
+  --white: #ffffff;
+}
+
```
Comment (RandilaP) [inline Gallery/styles.css:5]: here

**Review by RandilaP — CHANGES_REQUESTED** [review]

File: Gallery/styles.css [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+.container {
+  
+  display: inline-block;
+  justify-content: center;
+  align-items: center;
+  border:solid rgb(49, 214, 255);
+  width: 80%;
+}
+
+/* Gallery 1 */
```
Comment (RandilaP) [inline Gallery/styles.css:116]: When your having this kind of repetitive component use the same class. And if you want them to change separately add those separate changes using pseudo elements like :nth child. Try to learn about them and add it to the next issue.

File: Gallery/styles.css [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+}
+
+.container1-place-2 {
+  background: linear-gradient(white,rgb(182, 255, 233));
+  display: flex;
+  padding:10px
+}
+
+
+/* Gallery 2 */
```
Comment (RandilaP) [inline Gallery/styles.css:132]: this one

File: Gallery/styles.css [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+  margin-bottom: 20px;
+}
+
+.container2-place-2 {
+  background: linear-gradient(white,rgb(182, 255, 233));
+  display: flex;
+  padding:10px
+}
+
+/* Gallery 3 */
```
Comment (RandilaP) [inline Gallery/styles.css:147]: this one.

**Comment by pramodh123kit** [comment]
@RandilaP Removed the extra white spaces as requested. Please take a look

**Review by RandilaP — APPROVED** [review]

**Pull request merged**